spec/charts/count_chart/count_chart_spec.rb

```diff
@@ -175,24 +177,30 @@
   describe '#prepare_data' do
     it 'creates the prepared_data for simple keys' do
       chart = Charts::CountChart.new([3, 2], colors: ['x', 'o'], columns: 2)
-      expect(chart.prepared_data).to eq([
-                                          ['x', 'x'],
-                                          ['x', 'o'],
-                                          ['o']
-                                        ])
+      expect(chart.prepared_data).to eq(
+        [
+          ['x', 'x'],
+          ['x', 'o'],
+          ['o']
+        ]
+      )
     end
     it 'creates the prepared_data for complex keys' do
       chart = Charts::CountChart.new([2, 2], colors: ['#FF0000', '#00FF00'], columns: 2)
-      expect(chart.prepared_data).to eq([
-                                          ['#FF0000', '#FF0000'],
-                                          ['#00FF00', '#00FF00']
-                                        ])
+      expect(chart.prepared_data).to eq(
+        [
+          ['#FF0000', '#FF0000'],
+          ['#00FF00', '#00FF00']
+        ]
+      )
     end
     it 'default colors get assigned when no colors are specified' do
       chart = Charts::CountChart.new([1, 1, 1])
-      expect(chart.prepared_data).to eq([
-                                          ['#e41a1d', '#377eb9', '#4daf4b']
-                                        ])
+      expect(chart.prepared_data).to eq(
+        [
+          ['#e41a1d', '#377eb9', '#4daf4b']
+        ]
+      )
```

Comment:
So now you should be able to look at the ones that are pulled out like this, and determine if it is necessary to do so for readability, or having it on one line is still OK.